### DISCUSSION TITLE
Map and server info

### DISCUSSION THREAD

**Discussion by CodeAndRun** [saul/demofile#249] [Questions] [answered]
I know it is presented in demo header - https://developer.valvesoftware.com/wiki/DEM_Format#Demo_Header. 
Could I extract this with demofile?

**Comment by saul** [comment, marked as answer]
Hi

Yes you can - the property is `DemoFile#header`. See the docs: https://saul.github.io/demofile/classes/_demo_.demofile.html#header

Thanks
